### PR TITLE
Missing property Tag

### PR DESCRIPTION
**Pull request by Anipik**

### CODE REVIEW

**Comment by ViktorHofer** [comment]
I recommend to catch such errors early by consuming the package via a private feed in corefx. With that you can also test your changes on CI.

File: src/Microsoft.DotNet.GenFacades/build/Microsoft.DotNet.GenFacades.targets [hunk 1]
```diff
@@ -1,6 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
-<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (ViktorHofer) [inline src/Microsoft.DotNet.GenFacades/build/Microsoft.DotNet.GenFacades.targets]: Also remove these in other files?

Comment (Anipik) [reply src/Microsoft.DotNet.GenFacades/build/Microsoft.DotNet.GenFacades.targets]: There are no such lines in genfacades assembly.  do u want me to remove from other arcade projects ?

**Comment by Anipik** [comment]
> I recommend to catch such errors early by consuming the package via a private feed in corefx. With that you can also test your changes on CI.

I am testing with publishing to a local nuget feed and consuming the package from there but i think i missed to test with the final formatting

File: src/Microsoft.DotNet.GenFacades/build/Microsoft.DotNet.GenFacades.targets (hunk 1 above)
Comment (ViktorHofer) [reply src/Microsoft.DotNet.GenFacades/build/Microsoft.DotNet.GenFacades.targets]: Right, thanks.

**Pull request merged**